src/SFA.DAS.AssessorService.Application.Api/Services/StandardService.cs

```diff
@@ -203,6 +222,22 @@ private StandardCollation MapDataToStandardCollation(int standardId, IfaStandard
             };
         }
 
+        private List<string> MapIfaStandardOptions(IfaStandard ifaStandard)
```

Comment:
Could this method be put on the IfaStandard object?  

e.g.  ifaSrtandard.GetOptionsList()

Reply:
Agree - done